src/hooks/useSelectChain.ts

```diff
@@ -16,12 +17,17 @@ export default function useSelectChain() {
     async (targetChain: SupportedChainId) => {
       if (!connector) return
 
-      const connectionType = getConnection(connector).type
+      const connection = getConnection(connector)
+      const connectionType = connection.type
 
       try {
         dispatch(updateConnectionError({ connectionType, error: undefined }))
         await switchChain(connector, targetChain)
       } catch (error) {
+        if (didUserReject(connection, error)) {
+          return
+        }
+
         console.error('Failed to switch networks', error)
 
         dispatch(updateConnectionError({ connectionType, error: error.message }))
```

Comment:
What happens with this connection error? Is it still used?

Reply:
Yeah it still goes through if there is a connection error the toast still should pop up

Reply:
This gets picked up by our sentry logger that's the reason I kept it around

Reply:
If this is only here for Sentry, can you just explicitly call Sentry to report the exception? No need to go the roundabout way and add complexity.